restapi/src/main/java/io/stargate/web/docsapi/service/util/DocsApiUtils.java

```diff
@@ -296,13 +312,15 @@ public static boolean isRowOnPath(Row row, Iterable<String> pathIterable) {
 
       boolean pathSegment = target.contains(",");
```

Comment:
Not refactored as we talked yesterday? Is it not possible or? If not can we use 

`boolean pathSegment = COMMA_PATTERN.matches(target);` for the sake of being correct here..